from PyQt5 import QtWidgets,uic
from load.load_ventana_modelos_basicos import Load_ventana_modelos_basicos
from load.load_ventana_ocho import Load_ventana_ocho
class Load_ventana_principal(QtWidgets.QMainWindow):
    def __init__(self):
        super().__init__()
        
        #cargar la interfaz grafica
        uic.loadUi("interfaces/Ventana_principal.ui",self)
        
        #maximizar ventana
        self.showMaximized()

        self.actionBasico.triggered.connect(self.abrirVentanaBasicos)
        self.actionLangchain.triggered.connect(self.abrirVentanaLangchain)
        self.actionSalir.triggered.connect(self.cerrarVentana)
    
    def abrirVentanaBasicos(self):
        self.basicos=Load_ventana_modelos_basicos()
        self.basicos.exec_()
    def abrirVentanaLangchain(self):
        self.langchain=Load_ventana_ocho()
        self.langchain.exec_()
    def cerrarVentana(self):
        self.close()